run/test/specs/disappearing_message_checks_android.spec.ts

```diff
@@ -0,0 +1,477 @@
+import { androidIt } from "../../types/sessionIt";
+import {
+  DMTimeOption,
+  DisappearActions,
+  DisappearModes,
+} from "../../types/testing";
+import { sleepFor } from "./utils";
+import { newUser } from "./utils/create_account";
+import { newContact } from "./utils/create_contact";
+import { joinCommunity } from "./utils/join_community";
+import {
+  SupportedPlatformsType,
+  closeApp,
+  openAppTwoDevices,
+} from "./utils/open_app";
+import { setDisappearingMessage } from "./utils/set_disappearing_messages";
+
+async function disappearingImageMessage(platform: SupportedPlatformsType) {
+  const { device1, device2 } = await openAppTwoDevices(platform);
+  const testMessage = "Testing disappearing messages for images";
+  const time = "1 minute";
+  const mode: DisappearActions = "sent";
+  // Create user A and user B
+  const [userA, userB] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+  ]);
+  await newContact(platform, device1, userA, device2, userB);
+
+  await setDisappearingMessage(
+    platform,
+    device1,
+    ["1:1", "Disappear after send option", "1 minute"],
+    device2
+  );
+  // TODO FIX CONTROL MESSAGES ON ANDROID
```

Comment:
you should really have a way to have those somewhere in the report. so you have a summary of what's left.
Maybe skip the test entirely in the androidIt call? (this needs to  be built)